modules/3d/include/opencv2/3d.hpp

```diff
@@ -2523,15 +2523,15 @@ class CV_EXPORTS Octree {
     *
     * @param maxDepth The max depth of the Octree. The maxDepth > -1.
     */
-    explicit Octree(int maxDepth);
+    explicit Octree(size_t maxDepth);
```

Comment:
why change API w/o any good reason?

Reply:
Because I think maxDepth should be non-negative, and part of the subsequent code compares maxDepth to unsigned integers, I adopted this approach to avoid potential problems.

In addition, I forgot to change the corresponding explanation, which is my negligence.

Reply:
It's better to have `int` here since, among other reasons, it has better compatibility with our wrapper generators for Python and Java.